client/__tests__/e2e/e2eAnnotations.test.js

```diff
@@ -21,7 +21,7 @@ afterAll(async () => {
 
 describe.each([
   { withSubset: true, tag: "subset" },
-  { withSubset: false, tag: "whole" },
+  // { withSubset: false, tag: "whole" },
```

Comment:
```suggestion
  { withSubset: false, tag: "whole" },
```
Just re-enabling subset

Reply:
Oh thanks for catching that 😄 Enabled!